modules/osd-gcp-psc-firewall-prerequisites.adoc

```diff
@@ -0,0 +1,232 @@
+|===
++
+
+[NOTE]
+====
+Managed clusters require the enabling of telemetry to allow Red Hat to react more quickly to problems, better support the customers, and better understand how product upgrades impact clusters. For more information about how remote health monitoring data is used by Red Hat, see _About remote health monitoring_ in the _Additional resources_ section.
+====
+
+. Add the following {product-title} URLs to an allowlist:
++
+[cols="6,1,6",options="header"]
+|===
+|Domain | Port | Function
+
+|`mirror.openshift.com`
+|443
+|Used to access mirrored installation content and images. This site is also a source of release image signatures.
+
+|`api.openshift.com`
+|443
+|Used to check if updates are available for the cluster.
+|===
+
+. Add the following site reliability engineering (SRE) and management URLs to an allowlist:
++
+[cols="6,1,6",options="header"]
+|===
+|Domain | Port | Function
+
+|`api.pagerduty.com`
+|443
+|This alerting service is used by the in-cluster alertmanager to send alerts notifying Red{nbsp}Hat SRE of an event to take action on.
+
+|`events.pagerduty.com`
+|443
+|This alerting service is used by the in-cluster alertmanager to send alerts notifying Red{nbsp}Hat SRE of an event to take action on.
+
+|`api.deadmanssnitch.com`
+|443
+|Alerting service used by {product-title} to send periodic pings that indicate whether the cluster is available and running.
+
+|`nosnch.in`
+|443
+|Alerting service used by {product-title} to send periodic pings that indicate whether the cluster is available and running.
+
+|`*.osdsecuritylogs.splunkcloud.com`
+
+OR
+
+`inputs1.osdsecuritylogs.splunkcloud.com`
+
+`inputs2.osdsecuritylogs.splunkcloud.com`
+
+`inputs4.osdsecuritylogs.splunkcloud.com`
+
+`inputs5.osdsecuritylogs.splunkcloud.com`
+
+`inputs6.osdsecuritylogs.splunkcloud.com`
+
+`inputs7.osdsecuritylogs.splunkcloud.com`
+
+`inputs8.osdsecuritylogs.splunkcloud.com`
+
+`inputs9.osdsecuritylogs.splunkcloud.com`
+
+`inputs10.osdsecuritylogs.splunkcloud.com`
+
+`inputs11.osdsecuritylogs.splunkcloud.com`
+
+`inputs12.osdsecuritylogs.splunkcloud.com`
+
+`inputs13.osdsecuritylogs.splunkcloud.com`
+
+`inputs14.osdsecuritylogs.splunkcloud.com`
+
+`inputs15.osdsecuritylogs.splunkcloud.com`
+|9997
+|Used by the `splunk-forwarder-operator` as a logging forwarding endpoint to be used by Red{nbsp}Hat SRE for log-based alerting.
+
+|`http-inputs-osdsecuritylogs.splunkcloud.com`
+|443
+|Used by the `splunk-forwarder-operator` as a logging forwarding endpoint to be used by Red{nbsp}Hat SRE for log-based alerting.
+
+|`sftp.access.redhat.com` (Recommended)
+|22
+|The SFTP server used by `must-gather-operator` to upload diagnostic logs to help troubleshoot issues with the cluster.
+|===
+
+. Add the following URLs for the {GCP} API endpoints to an allowlist:
++
+[cols="6,1,6",options="header"]
+|===
+|Domain | Port | Function
+
+| `accounts.google.com`
+| 443
+| Used to access your GCP account.
+
+|`*.googleapis.com`
+
+OR
```

Comment:
We need to add a two other ones to this list, sorry for the miss:
```
dns.googleapis.com
iamcredentials.googleapis.com
```

Reply:
Got it, thanks for catching. Will update today.